javaslang/src/main/java/javaslang/collection/Vector.java

```diff
@@ -584,24 +580,16 @@ private Vector(BitMappedTrie<T> trie) {
     }
 
     @Override
-    public Vector<Vector<T>> combinations() {
-        return rangeClosed(0, length()).map(this::combinations).flatMap(Function.identity());
-    }
+    public Vector<Vector<T>> combinations() { return rangeClosed(0, length()).map(this::combinations).flatMap(Function.identity()); }
```

Comment:
I formatted these manually, the formatter should keep them intact (mine does).
I wanted the formatting to reflect how it would look in e.g. `Scala` or in a Java 8 lambda: very simple methods in the same line, complex ones on multiple, to increase SNR.